src/main/java/edu/ucsb/cs156/dining/controllers/MenuItemReviewController.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * This is a REST controller for MenuItemReview
+ */
+
+@Tag(name = "MenuItemReview")
+@RequestMapping("/api/menuitemreviews")
+@RestController
+@Slf4j
+public class MenuItemReviewController extends ApiController {
+
+    @Autowired
+    MenuItemReviewRepository menuItemReviewRepository;
+
+    @Autowired
+    MenuItemRepository menuItemRepository;
+    
+    /**
+     * Create a new menu item review -> all users
+     */
+
+    @Operation(summary= "Create a new menu item review")
+    @PreAuthorize("hasRole('ROLE_USER')")
+    @PostMapping("/post")
+    public MenuItemReview postMenuItemReview(
+            @Parameter(name="studentUserId") @RequestParam long studentUserId,
+            @Parameter(name="itemId") @RequestParam long itemId,
+            @Parameter(name="itemServedDate", description="date (in iso format, e.g. YYYY-mm-ddTHH:MM:SS") @RequestParam("itemServedDate") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime itemServedDate,
+            @Parameter(name="status") @RequestParam String status,
+            @Parameter(name="moderatorUserId") @RequestParam long moderatorUserId,
+            @Parameter(name="moderatorComments") @RequestParam String moderatorComments,
+            @Parameter(name="createdDate", description="date (in iso format, e.g. YYYY-mm-ddTHH:MM:SS") @RequestParam("createdDate") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime createdDate,
+            @Parameter(name="lastEditedDate", description="date (in iso format, e.g. YYYY-mm-ddTHH:MM:SS") @RequestParam("lastEditedDate") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime lastEditedDate)
+    
+            throws JsonProcessingException {
+
+                MenuItem menuItem = menuItemRepository.findById(itemId).orElseThrow(() -> 
+                    new ResponseStatusException(HttpStatus.NOT_FOUND, "MenuItem with ID " + itemId + " not found"));
+
+        log.info("itemServedDate={}", itemServedDate);
+
+        MenuItemReview menuItemReview = new MenuItemReview();
+        menuItemReview.setStudentUserId(studentUserId);
+        menuItemReview.setMenuItem(menuItem);
+        menuItemReview.setItemServedDate(itemServedDate);
+        menuItemReview.setStatus(status);
+        menuItemReview.setModeratorUserId(moderatorUserId);
+        menuItemReview.setModeratorComments(moderatorComments);
+        menuItemReview.setCreatedDate(createdDate);
+        menuItemReview.setLastEditedDate(lastEditedDate);
+
+        MenuItemReview savedMenuItemReview = menuItemReviewRepository.save(menuItemReview);
+
+        return savedMenuItemReview;
+    }
+
+
+    /**
+     * List all menu item reviews -> ADMIN ONLY
+     */
+    @Operation(summary= "List all menu item reviews")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @GetMapping("/all")
+    public Iterable<MenuItemReview> allMenuItemReviewAdminOnly() {
+        Iterable<MenuItemReview> reviews = menuItemReviewRepository.findAll();
+        return reviews;
+    }
```

Comment:
What's the use case for this?

I do understand why an admin would need to list all reviews that have the status "Awaiting Moderation". But when would they just need to see a list of all the reviews?